src/components/tooltip/_tooltip.scss

```diff
@@ -140,13 +142,12 @@
     &:before {
       @include typescale('zeta');
       @include layer('overlay');
-      max-width: rem(240px);
-      padding: $spacing-lg;
+      max-width: rem(170px);
+      padding: rem(10px) $spacing-xs;
```

Comment:
Might be a question for design but can `rem(10px)` be tweaked up or down to fit into our spacing scale?

Reply:
It should be 8px